lib/smart_app_launch/token_introspection_request_group.rb

```diff
@@ -0,0 +1,121 @@
+require_relative 'token_exchange_test'
+require_relative 'token_refresh_body_test'
+require_relative 'well_known_endpoint_test'
+require_relative 'standalone_launch_group'
+
+module SMARTAppLaunch
+  class TokenIntrospectionRequestGroup < Inferno::TestGroup
+    title 'Token Introspection Request'
```

Comment:
In looking at this in the interface, i wonder if we should call this first test `Issue Introspection Request` and the second `Validate Introspection Response`.  Its optimized a bit more towards thinking about the larger test group as 2 steps, and is still a bit imprecise because this is doing more than just issuing a request (it is checking some parts of the response).  But I think it might be less confusing like that?

We could also have 2 groups, 1 that is something like 'Validate Introspection Endpoint' and another that is 'Validate Introspection Response Only', and in the first group it imports the 2 tests in the 'Validate Introspection Response Only' tests, so it is always an either-or thing.  But I think probably going with our initial approach is fine because either way I could see people getting confused.